frontend/src/Explorer/View/Common.purs

```diff
@@ -295,8 +295,8 @@ noData = "--"
 currencyCSSClass :: Maybe CCurrency -> String
 currencyCSSClass mCurrency =
   case mCurrency of
-      Just ADA -> " ada bg-ada-dark"
-      Just USD -> " usd bg-usd-dark"
+      Just ADA -> " currency ada bg-ada-dark"
+      Just USD -> " currency usd bg-usd-dark"
```

Comment:
long term it would be great to have some newtype wrapper for CSS instead of plane String. Something like:
```
type CSSClasses = Array CSSClass
newtype CSSClass = CSSClass String

currencyCSSClass :: Maybe CCurrency -> CSSClasses
```
something like this. An immediate step would be just using:
```
type CSSClass = String
```
and it will help at least to document a code